fix error in rect-geo (#2331)
cpinitiative/usaco-guide: content/2_Bronze/Rect_Geo.mdx

involve finding the overlapping area of two or more rectangles based on their coordinate points, 
or determining whether two rectangles intersect. Here, we'll discuss these formulas. 

<Info title="Coordinates">

> Rectangles can be represented by $2$ points each, the top right and the bottom left. We'll call these points $tr_a$ and $tr_b$, and $bl_a$ and $bl_b$, for rectangles $a$ and $b$. In examples with just one rectangle, we'll omit it to $tr_x$ and $tr_y$.

Reviewer: ```suggestion
A rectangle can be represented with $2$ points: its top right corner and bottom left corner. We'll label these points $tr$ (top right) and $br$ (bottom right).
```

		```java
		int area(int bl_x, int bl_y, int tr_x, int tr_y) {
			int length = tr_y - bl_y;
			int width = tr_x - bl_x;
			return length * width;
		}
		```
	</JavaSection>
</LanguageSection>

> ### Checking if rectangles intersect

Reviewer: ```suggestion
### Checking if two rectangles intersect
```

> There are only two cases where the rectangles do not intersect: 

Reviewer: ```suggestion
Given two rectangles $a$ and $b$, there are only two cases where they do not intersect: 
```

We'll assume that the shape formed by the intersection of two rectangles is itself a rectangle.

First, we'll find this rectangle's $x$ and $y$ coordinates. 

> $x$ = $\min(tr_{a_x}, tr_{b_x}) - \max(bl_{a_x}, bl_{b_x})$. 
> $y$ = $\min(tr_{a_y}, tr_{b_y}) - \max(bl_{a_y}, bl_{b_y})$.

Reviewer: ```suggestion
$x = \min(tr_{a_x}, tr_{b_x}) - \max(bl_{a_x}, bl_{b_x})$. 
$y = \min(tr_{a_y}, tr_{b_y}) - \max(bl_{a_y}, bl_{b_y})$.
```
